bouncer/util.py

```diff
@@ -70,4 +101,35 @@ def parse_document(document):
             _("The annotation has an invalid document URI"),
             "uri_not_a_string")
 
-    return (annotation_id, document_uri)
+    return {
+            "annotation_id": annotation_id,
+            "document_uri": document_uri,
+            "can_reveal_metadata": can_reveal_metadata,
+            "quote": quote,
+            "text": text
+            }
+
+
+def make_pretty_url(url):
```

Comment:
Similar to another comment, `make_` is not a very common prefix. The old function name was good enough IMO (except for the underscore describing that it's a private function). Why not just `pretty_url`? Other good alternatives might be `get_pretty_url` or `generate_pretty_url`.